plugins/org.locationtech.udig.tool.measure/src/org/locationtech/udig/tool/measure/internal/messages_ru.properties

```diff
@@ -1,2 +1,2 @@
 DistanceTool_error=\u041d\u0435\u0432\u043e\u0437\u043c\u043e\u0436\u043d\u043e \u0432\u044b\u0447\u0438\u0441\u043b\u0438\u0442\u044c \u0440\u0430\u0441\u0441\u0442\u043e\u044f\u043d\u0438\u0435
-DistanceTool_distance=\u0420\u0430\u0441\u0441\u0442\u043e\u044f\u043d\u0438\u0435 \= {0}
\ No newline at end of file
+DistanceTool_distance=\u0420\u0430\u0441\u0441\u0442\u043e\u044f\u043d\u0438\u0435 \= {0} (\u043F\u043E\u0441\u043B\u0435\u0434\u043D\u0438\u0439 \u0441\u0435\u0433\u043C\u0435\u043D\u0442 \= {1})
```

Comment:
Cannot check this ;)

Reply:
Tested new functionality and it works as expected. Awesome to allo scrolling/zooming while tools is still active.

Reply:
> 
> 
> Cannot check this ;)

My either I just rely in google translation!!